src/core/Akka.Remote.TestKit/MultiNodeSpec.cs

```diff
@@ -508,7 +508,6 @@ public int InitialParticipants
         /// </summary>
         public void RunOn(Action thunk, params RoleName[] nodes)
         {
-            if (nodes.Length == 0) throw new ArgumentException("No node given to run on.");
```

Comment:
Added this to allow a `RunOn` statement to execute for zero nodes if needed, which we do in some cases inside the `StressSpec`, since that spec is dynamic and runs for a configurable number of nodes at each stage.